velox/functions/prestosql/aggregates/SimpleNumericAggregate.h

```diff
@@ -126,12 +126,6 @@ class SimpleNumericAggregate : public exec::Aggregate {
         updateNonNullValue<tableHasNulls, TData>(
             groups[i], TData(decoded.valueAt<TValue>(i)), updateSingleValue);
       });
-    } else if (decoded.isIdentityMapping() && !std::is_same_v<TValue, bool>) {
```

Comment:
@mbasmanova raised a concern that this would cause performance regression.  I agree we should pay attention on that.  Can we put this back?

Reply:
CMIIW, the perf regression is introduced because valueAt<> ends up checking isIdentityMapping for each element of the vector and the we want to avoid that.

Reply:
Yes I think so